tests/remote_resource/test_requests.py

```diff
@@ -130,9 +159,21 @@ class RouteRetrievalTestCase(TestCase):
     def setUp(self):
         self.gateway_root_path = 'http://mp-gateway.zerocater.com'
         self.mappings = {
-            '/users/{id}': {
+            '/users': {
+                'domain': 'https://mp-users.zerocater.com',
+                'resource_type': 'User'
+            },
```

Comment:
You can remove the `/users` and `/vendors` gateway routes. I believe the gateway will only have one route associated with each resource_type.